docs/solutions/rhel74_image/README.md

```diff
@@ -0,0 +1,40 @@
+This example provides method to generate a RHEL 7.4 image for use by both VM and BM shapes.
+There are several prerequisites for using this process:
+
```

Comment:
If there's a relevant white paper to go with this, would make sense to link to it.